hexa/databases/utils.py

```diff
@@ -0,0 +1,86 @@
+from typing import Dict, List, Tuple
+
+import psycopg2
+from psycopg2 import sql
+
+from hexa.plugins.connector_postgresql.models import Database
+
+
+def get_database_tables_summary(database: Database, table=None, limit_per_table=4):
```

Comment:
It's also better to handle the pagination of the result in this function than in the resolver

Reply:
`get_database_definition` is more appropriate
I think it's better to hide the implementation detail of the database layer and just pass a workspace instead of a database from the connector_postgresql (this will not be used in the future I think)

Reply:
the `def get_table_data(workspace: Workspace, tableName: str, rows: int=4)` has to be a different function

Reply:
It's also better I think to handle the unique table case in the `get_table_definition` (where you would return {"name": "My_table", "columns": [], "row_count": 10}